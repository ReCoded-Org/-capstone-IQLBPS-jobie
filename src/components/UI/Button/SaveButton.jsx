function SaveButton(){
  return(
    <div className="flex justify-end p-10 font-inter small:flex-col small:space-y-1 small:items-end  ">
      <button type="button" className=" py-2 font-bold bg-accent px-5 rounded-3xl text-white transition ease-in-out delay-150 hover:-translate-y-1 hover:scale-110  duration-300 small:w-24 small:m-0  small:text-xs">Save</button>
      <button type="button" className="ml-4 py-2 font-bold bg-blue-200 px-5 rounded-3xl text-dark-gray transition ease-in-out delay-150 hover:-translate-y-1 hover:scale-110  duration-300 small:w-24 small:m-0 small:text-xs">Cancel</button>
    </div>
  )
}
export default SaveButton;