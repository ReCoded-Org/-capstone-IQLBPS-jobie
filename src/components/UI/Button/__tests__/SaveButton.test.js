import React from "react";
import renderer from "react-test-renderer";
import SaveButton from "../SaveButton";
it("renders Custom button correctly", () => {
  const tree = renderer.create(<SaveButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
