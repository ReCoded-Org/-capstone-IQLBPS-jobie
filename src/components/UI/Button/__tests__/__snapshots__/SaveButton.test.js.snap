// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Custom button correctly 1`] = `
<div
  className="flex justify-end p-10 font-inter small:flex-col small:space-y-1 small:items-end  "
>
  <button
    className=" py-2 font-bold bg-accent px-5 rounded-3xl text-white transition ease-in-out delay-150 hover:-translate-y-1 hover:scale-110  duration-300 small:w-24 small:m-0  small:text-xs"
    type="button"
  >
    Save
  </button>
  <button
    className="ml-4 py-2 font-bold bg-blue-200 px-5 rounded-3xl text-dark-gray transition ease-in-out delay-150 hover:-translate-y-1 hover:scale-110  duration-300 small:w-24 small:m-0 small:text-xs"
    type="button"
  >
    Cancel
  </button>
</div>
`;
